maestro-cli/src/main/java/maestro/cli/command/StartDeviceCommand.kt

```diff
@@ -0,0 +1,70 @@
+package maestro.cli.command
+
+import maestro.cli.CliError
+import maestro.cli.device.DeviceCreateUtil
+import maestro.cli.device.DeviceService
+import maestro.cli.device.Platform
+import maestro.cli.report.TestDebugReporter
+import maestro.cli.util.DeviceConfigAndroid
+import maestro.cli.util.DeviceConfigIos
+import maestro.cli.util.PrintUtils
+import picocli.CommandLine
+import java.util.concurrent.Callable
+
+@CommandLine.Command(
+    name = "start-device",
+    description = [
+        "Starts or creates an iOS Simulator or Android Emulator similar to the ones on Maestro Cloud",
+        "Supported device types: iPhone11 (iOS), Pixel 6 (Android)",
+    ]
+)
+class StartDeviceCommand : Callable<Int> {
+
+    @CommandLine.Option(
+        order = 0,
+        names = ["--platform"],
+        required = true,
+        description = ["Platforms: android, ios"],
+    )
+    private lateinit var platform: String
+
+    @CommandLine.Option(
+        order = 1,
+        names = ["--os-version"],
+        description = ["OS version to use:", "iOS: 15, 16", "Android: 28, 29, 30, 31, 33"],
+    )
+    private lateinit var osVersion: String
+
+    @CommandLine.Option(
+        order = 3,
+        names = ["--force-create"],
+        description = ["Will override existing device if it already exists"],
+    )
+    private var forceCreate: Boolean = false
+
+
+    override fun call(): Int {
+        TestDebugReporter.install(null)
+
+        val p = Platform.fromString(platform) ?: throw CliError("Unsupported platform $platform. Please specify one of: android, ios")
+
+        // default OS version
+        if (!::osVersion.isInitialized) {
+            osVersion = when (p) {
+                Platform.IOS -> DeviceConfigIos.defaultVersion.toString()
+                Platform.ANDROID -> DeviceConfigAndroid.defaultVersion.toString()
+                else -> ""
+            }
+        }
+        val o = osVersion.toIntOrNull()
+
+        DeviceCreateUtil.getOrCreateDevice(p, o, forceCreate).let {
+            PrintUtils.message(if (p == Platform.IOS) "Launching simulator..." else "Launching emulator...")
+            DeviceService.startDevice(it)
+        }
```

Comment:
What is the error message here when trying to start an iOS device from a Windows/Linux machine?